src/App.js

```diff
@@ -0,0 +1,152 @@
+import { useState } from "react";
+
+function Square({ value, onSquareClick, squareColor }) {
+  return (
+    <button
+      className="square"
+      onClick={onSquareClick}
+      style={{
+        backgroundColor: squareColor ? "white" : "yellow"
+      }}
+    >
+      {value}
+    </button>
+  );
+}
+
+function Board({ xIsNext, squares, onPlay }) {
+  const winner = calculateWinner(squares);
+  console.log(winner);
+  const row = [];
+  let u = 0;
+
+  function handleClick(i) {
+    if (squares[i] || calculateWinner(squares)) {
+      return;
+    }
+    const nextSquares = squares.slice();
+    if (xIsNext) {
+      nextSquares[i] = "X";
+    } else {
+      nextSquares[i] = "O";
+    }
+    onPlay(nextSquares);
+  }
+
+  let status;
+  if (winner) {
+    status = "Winner: " + winner[0];
+  } else if (!squares.includes(null) & !winner) {
+    status = "Game Over: Draw";
+  } else {
+    status = "Next player: " + (xIsNext ? "X" : "O");
+  }
+
+  for (let t = 0; t < 3; t++) {
+    const col = [];
+    for (let s = 0; s < 3; s++) {
+      let defaultColor = true;
+      if (winner) {
+        if (winner[1].includes(u)) {
+          defaultColor = false;
+        }
+      }
+      col.push(
+        <Square
+          key={u}
+          value={squares[3 * t + s]}
+          onSquareClick={() => handleClick(3 * t + s)}
```

Comment:
```suggestion
          value={squares[getSquareIndex(t, s)]}
          onSquareClick={() => handleClick(getSquareIndex(t, s))}
```
We can create a function for calculating square index and reuse it